### PR TITLE
Proper way to undo a sudo context

### PR DESCRIPTION
**Pull request by yohanboniface**
Before, we were doing a sudo of the ssh user on itself, now we properly remove any sudo.

### CODE REVIEW

File: tests/unit/test_command.py [hunk 1]
```diff
@@ -54,3 +54,21 @@ def test_ls_without_all(patch_client):
...
+def test_sudo_user(patch_client):
+    with usine.sudo(user="me"):
+        assert usine.run('whoami') == \
+            "sudo --set-home --preserve-env --user=me --login sh -c $'whoami'"
+
+
+def test_unsudo(patch_client):
+    with usine.sudo():
+        with usine.unsudo():
+            assert usine.run('whoami') == "sh -c $'whoami'"
```
Comment (davidbgk) [inline tests/unit/test_command.py:74]: Is `test_unsudo_user` relevant? Do we want to preserve the initial user?

**Review by davidbgk — APPROVED** [review]

File: tests/unit/test_command.py (hunk 1 above)
Comment (yohanboniface) [reply tests/unit/test_command.py:74]: `unsudo` should totally remove the `sudo` context, and its parameters (`user` including).

Does that answer? :)

Comment (yohanboniface) [reply tests/unit/test_command.py:74]: It's suppose to be used inside a `sudo` context to temporarily run a command without this context.

Comment (davidbgk) [reply tests/unit/test_command.py:74]: Yup, 👌

**Review by vinyll — APPROVED** [review]

**Pull request merged**